gradle-plugins/build.gradle.kts

```diff
@@ -22,9 +22,9 @@ dependencies {
   implementation("com.google.guava:guava:30.1.1-jre")
   implementation("net.bytebuddy:byte-buddy-gradle-plugin:1.11.2")
 
-  implementation("io.opentelemetry.instrumentation:opentelemetry-instrumentation-api:1.5.0-alpha-SNAPSHOT")
-  implementation("io.opentelemetry.javaagent:opentelemetry-muzzle:1.5.0-alpha-SNAPSHOT")
-  implementation("io.opentelemetry.javaagent:opentelemetry-javaagent-extension-api:1.5.0-alpha-SNAPSHOT")
+  implementation("io.opentelemetry.instrumentation:opentelemetry-instrumentation-api:1.6.0-alpha")
```

Comment:
Did you mean 1.5.0? There is no 1.6 yet :)

Reply:
oops 😬

I will send separate PR to add building gradle-plugins to the build files